src/Command/DispatchCommand.php

```diff
@@ -32,6 +32,13 @@ final class DispatchCommand extends AbstractNeedApplyCommand
     private const LABEL_NOTHING_CHANGED = 'Nothing to be changed.';
     private const FILES_DIR = 'project';
 
+    /** @var string[] */
+    private const HOOK_URLS_TO_BE_DELETED = [
+        'https://api.codacy.com',
```

Comment:
I tested this with this hook and `admin-bundle` and it removes it like expected 👍